src/features/More/more.model.ts

```diff
@@ -0,0 +1,8 @@
+interface Category {
+  title: string;
+  id: string;
+  is_parent_category: boolean;
+  //children: []; // TODO: can't assing Category[] type...
```

Comment:
what is this?

Reply:
I was thinking about the future structure of the category model when certain categories have sub-categories.